spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -39,6 +44,7 @@
 
 # Update manager process statuses
 NO_STATUS = __version__
+UPDATING_UPDATER = _("Updating Spyder-updater package")
```

Comment:
```suggestion
UPDATING_UPDATER = _("Updating Spyder-updater")
```

I didn't realize this message is shown in the status bar. In that case I think it should be it shorter.